src/org/labkey/remoteapi/query/SelectRowsCommand.java

```diff
@@ -187,12 +186,15 @@ protected SelectRowsResponse createResponse(String text, int status, String cont
     @Override
     public Map<String, Object> getParameters()
     {
-        Map<String, Object> params = new HashMap<>();
+        Map<String, Object> params = super.getParameters();
```

Comment:
nit: This may be more than you want to bite off in this PR but I compared the parameters we process here versus the parameters we process in the JavaScript equivalent and there some dispartities.

https://github.com/LabKey/labkey-api-js/blob/b2448eb9d/src/labkey/query/SelectRows.ts#L123

Ones that may be useful for Java clients are `includeTotalCount`, `includeMetadata`, and `ignoreFilter`.

Reply:
Done